src/view/navigation/index.tsx

```diff
@@ -15,13 +15,17 @@ import drawerActions from '../../store/modules/drawer/actions';
 import searchBarActions from '../../store/modules/searchBar/actions';
 import navActions from '../../store/modules/nav/actions';
 import store from '../../store';
+import { Farmer } from '../../store/modules/farmer/types';
+import { getActiveFarmer } from '../../store/modules/farmer/selectors'; 
 
 interface StorePropsType {
   nav: NavigationState;
   headerShown: boolean;
+  routeName: string;
   routeType: PageType | undefined;
   searchBarShown: boolean;
   searchPlaceholder: string | undefined;
+  currentFarmer: Farmer; 
```

Comment:
Should we add `currentMilkEntry` here too? to update the header of the page to the entry we clicked on